test/handshake/errors.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: circt-opt %s --split-input-file --verify-diagnostics
+
```

Comment:
Maybe this test case should locate in test/Dialect/Handshake/ ?

Reply:
That's better, I'll move it.